overrides/home.html

```diff
@@ -4,7 +4,7 @@
 <div class="md-container">
   <section class="home-container">
     <div class="animated-gradient">
-      <img src="images/logo/rgb/knative-logo-rgb.png" />
+        <img src="images/logo/rgb/knative-logo-rgb.png" style="display: none;"/>
```

Comment:
let's keep the indentation like it was before here

Reply:
Done with the changes